versions/1.21.5/src/main/java/at/hannibal2/skyhanni/utils/compat/OrdereredTextUtils.kt

```diff
@@ -93,24 +93,31 @@ object OrderedTextUtils {
 
         return OrderedText { visitor ->
             var lastStyle = Style.EMPTY
-            var wasLastStyle = false
-
-            for (char in legacyString ?: "") {
-
-                if (char == '§') {
-                    wasLastStyle = true
-                } else if (wasLastStyle) {
-
-                    val formatting = Formatting.byCode(char)
+            val str = legacyString ?: ""
+            var i = 0
```

Comment:
You should probably name this something descriptive, generally avoid single letter variable names
```suggestion
            var index = 0
```